src/flashbar/__tests__/collapsible.test.tsx

```diff
@@ -174,169 +185,185 @@ describe('Collapsible Flashbar', () => {
+        const customLabels = {
+          errorIconAriaLabel: 'Custom error ARIA label',
+          successIconAriaLabel: 'Custom success ARIA label',
+          infoIconAriaLabel: 'Custom info ARIA label',
+          inProgressIconAriaLabel: 'Custom progress ARIA label',
+          warningIconAriaLabel: 'Custom warning ARIA label',
+        };
+        const flashbar = renderFlashbar({ i18nStrings: { ...customLabels } });
+        const innerCounter = findInnerCounterElement(flashbar);
+        for (const ariaLabel of Object.values(customLabels)) {
+          expect(innerCounter!.querySelector(`[aria-label="${ariaLabel}"]`)).toBeTruthy();
+          expect(innerCounter!.querySelector(`[title="${ariaLabel}"]`)).toBeTruthy();
+        }
+      });
+
+      test.each([['success'], ['error'], ['info'], ['warning'], ['in-progress']] as FlashbarProps.Type[][])(
+        'item icon has aria-label from i18nStrings when no statusIconAriaLabel provided: type %s',
+        type => {
+          const wrapper = renderFlashbar({
+            i18nStrings: {
+              successIconAriaLabel: 'success',
+              errorIconAriaLabel: 'error',
+              infoIconAriaLabel: 'info',
+              warningIconAriaLabel: 'warning',
+              inProgressIconAriaLabel: 'in-progress',
+            },
+            items: [
+              {
+                header: 'The header',
+                content: 'The content',
+                type: type === 'in-progress' ? 'info' : type,
+                loading: type === 'in-progress',
+              },
+            ],
+          });
+
+          expect(wrapper.findItems()[0].find('[role="img"]')?.getElement()).toHaveAccessibleName(type);
+        }
+      );
+    });
 
-  describe('Sticky', () => {
-    it('scrolls the button into view when collapsing', () => {
-      scrollElementIntoViewMock.mockClear();
-      const flashbar = renderFlashbar();
-      findNotificationBar(flashbar)!.click();
-      findNotificationBar(flashbar)!.click();
-      expect(scrollElementIntoViewMock).toHaveBeenCalledTimes(1);
+    describe('Sticky', () => {
+      test('scrolls the button into view when collapsing', () => {
+        scrollElementIntoViewMock.mockClear();
+        const button = renderFlashbar().findToggleButton()!;
+        button.click(); // Expand
+        button.click(); // Collapse
+        expect(scrollElementIntoViewMock).toHaveBeenCalledTimes(1);
+      });
     });
   });
+
+  test('dismisses items', () => {
+    // Test this feature only without animations because TransitionGroup delays item removals by one frame.
+    // Customers should disable animations in their tests too:
+    // https://cloudscape.design/foundation/visual-foundation/motion/#implementation
+    disableMotion(true);
+    testFlashDismissal({ stackItems: true });
+  });
+
+  test('produces the same visual HMTL output with motion enabled and disabled', () => {
```

Comment:
This test is less relevant after last commit, but it helped me catch that if using `<TransitionGroup>` without `component={null}`, it adds a wrapping div. This could actually break styles, eventually.

Reply:
When using with `removeInvisibleAttributes` the test is not very trust worthy.

Feel free to keep it, but I do not find it very useful